scheduler-utils/src/locate.js

```diff
@@ -29,7 +29,13 @@ export function locateWith ({
            */
           if (schedulerHint) {
             const byOwner = await cache.getByOwner(schedulerHint)
-            if (byOwner) return byOwner
+            if (byOwner) {
+              return {
+                url: byOwner.url,
+                owner: byOwner.owner,
```

Comment:
If the `getByOwner` stub in the `locateWith.should use the scheduler hint and use the cached owner` test was corrected to return the real shape, it would catch this issue.

Reply:
That's my point -- the existing test is using the wrong shape (which is how this bug got in, in the first place). The test ought to return `{ url, address, ttl }`, since that is the actual schema of the `byOwner` cache